### DISCUSSION TITLE
It is estimated lightning kills how many people a year worldwide?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#603] [Galaxy Trivia] [answered]
*answer in binary*

**Comment by peterbaccaro** [comment]
According to National Geographic, about **11111010000** people are killed annually worldwide by lightning.

**Comment by umlaeute** [comment, marked as answer]
[`11111010000`](https://www.nationalgeographic.com/environment/article/lightning) (although this was in 2009, probably the probability has changed with climate change getting traction)
